auza00-sp

auzkya opened the pull request:
First commit
- Neresponzivní grafika
- Přidávání a mazání spotu
- Like/unlike spot
- Přidání komentáře

auzkya committed:
auza00-sp

First commit

nvbach91 commented:
Prace neni odevzdana

nvbach91 reviewed (changes requested):
missing app URL

auzkya committed:
Delete www/auza00/sp2/database.sql

auzkya committed:
Delete www/auza00/sp2/utils directory

auzkya committed:
2nd commit

- google oAuth login
- fixed erros

To do:
- facebook login
- admin page
- clean code

auzkya commented:
Dobrý večer,
moc se omlouvám, z nějakého důvodu jsem měl zafixováno, že projekt stačí odevzdat do půlnoci před obhajobou.
Na dodělaní projektu intenzivně pracuji, každopádně ještě mi bohužel zbývá přihlášení přes Facebook, stránka pro Admina (na mazání jakéhokoliv spotu) a pročistit kód.
Odkaz na samotný web: https://iknowaspot.eu/

auzkya committed:
3rd commit

+ Login through Facebook working
+ Repaired problems with login through Google
+ Spots counter on the top of the map

auzkya committed:
4th commit

+ responsive design on mobile devices
+ working commenting and like/unlike on mobile devices

auzkya committed:
5th commit

small changes - scrolling from bottom etc

nvbach91 reviewed (changes requested):
- databazove rozhrani (OOP class pro kazdou tabulku) aspon pro uzivatele
- REST API
  - endpoint: GET /api/v1/spots?category=1 JSON
  - endpoint: GET /api/v1/comments?spot_id=15 JSON
- stranka seznam spotu /spots.php administrator only
- Cross site scripting `<script>` to text
- nerefreshovat stranku po pridavani komentare

auzkya committed:
Delete www/auza00/sp2 directory

Reuploading all files.

auzkya committed:
Reupload, nyní OOP class pro každou tabulku

auzkya committed:
nahrávání souborů

předtím se z nějakého důvodu nenahrály všechny soubory

auzkya committed:
Update DatabaseConnection.php

auzkya committed:
Delete www/auza00/sp2/js/comment-system.js

auzkya committed:
Delete www/auza00/sp2/js/functions.js

auzkya committed:
Delete www/auza00/sp2/js/liking-system.js

auzkya committed:
6th commit

opraveno:
 - nerefreshovat stranku po pridavani komentare
 - refresh mych spotu okamzite, nikoliv po celkovem restartu stranky (kvuli poctum liku)
 - pridani stranky /admin pro seznam spotu (pouze pro admina)
 - vymazani fotky ze serveru spolu se spotem

auzkya committed:
Update DatabaseConnection.php

auzkya committed:
7th commit

přidán režim je hezky/prší

auzkya committed:
8th commit

Některé funkce nefungovaly na mobilu (především komentování a likování)
Dobře přečtitelné datum a čas

auzkya committed:
8th commit

Někde datum fungovalo různě, opraveno
Přidáno datum ke spotu

auzkya committed:
minor issue fixed

nvbach91 commented:
- PULL REQUEST
- delete spot
- htmlspecialchars
- samostatny soubor pro konfiguracni udaje pro pripojeni do databaze

auzkya committed:
Delete comment directory

auzkya committed:
Delete admin directory

auzkya committed:
Delete css directory

auzkya committed:
Delete db directory

auzkya committed:
Delete img directory

auzkya committed:
Delete js directory

auzkya committed:
Delete like directory

auzkya committed:
Delete preset directory

auzkya committed:
Delete spot-img directory

auzkya committed:
Delete spot directory

auzkya committed:
Delete sql directory

auzkya committed:
Delete user directory

auzkya committed:
Delete www directory

auzkya committed:
Delete .gitignore

auzkya committed:
Delete LICENSE

auzkya committed:
Delete README.md

auzkya committed:
Delete index.php

auzkya committed:
Delete to-do.txt

auzkya committed:
9th commit

PULL REQUEST
htmlspecialchars
samostatny soubor pro konfiguracni udaje pro pripojeni do databaze

auzkya committed:
Merge pull request #216 from nvbach91/auza00-sp-1

auza00-sp

auzkya committed:
Delete www/auza00/sp/sql/i_know_a_spot.sql

auzkya committed:
Final commit

SQL database exported in folder 'sql'
